src/components/ContactImportationModal/index.jsx

```diff
@@ -47,15 +47,19 @@ class ContactImportationModal extends React.Component {
     this.updateImportation(runningImportation)
 
     const finishedImportation = await finishedImportationPromise
-
     if (finishedImportation.status === Status.COMPLETE_SUCCESS) {
       const created = Importation.created(finishedImportation)
       const updated = Importation.updated(finishedImportation)
+      const stringCreated =
+        created > 0 ? t('importation.creation', created) : ''
+      const stringUpdated =
+        updated > 0
+          ? created > 0
+            ? `, ${t('importation.update', updated)}`
+            : `${t('importation.update', updated)}`
+          : ''
```

Comment:
Not super fond of double ternaries, but I'm having a hard time coming up with a good alternative.